test/steps/general/3_then.js

```diff
@@ -35,13 +34,29 @@ export function itShouldReturnTheResult() {
 export function itShouldThrowValidationError() {
 	const { testFunction } = this.test.ctx;
 	const message = getFirstQuotedString(this.test.title);
-	return testFunction.should.throw(new ValidationError(message));
+	testFunction.should.throw().and.is.instanceOf(Error);
+	testFunction.should
+		.throw()
+		.and.has.property('name')
+		.which.equal('ValidationError');
+	return testFunction.should
+		.throw()
+		.and.has.property('message')
+		.which.include(message);
```

Comment:
All these `testFunction.should.throw()`s seem redundant. It's awkward, but I think it would be better to store that in a variable and make further assertions against it.